packages/css-engine/src/core/to-value.ts

```diff
@@ -91,12 +91,21 @@ export const toValue = (
   }
 
   if (value.type === "layers") {
-    return value.value
-      .map((value) => toValue(value, transformValue))
+    const valueString = value.value
+      .map((layer) => toValue(layer, transformValue))
+      .filter((value) => value !== "none")
```

Comment:
Why filter "none"?
If value accepts none it must be rendered. If not then no such style should occure at all

Reply:
@JayaKrishnaNamburu ?

Reply:
In that case this must be covered in tests. And I still think that relying on none is mistake, you can just check here that tuple is hidden and don't call toValue

Reply:
Also to remove all this logic just return `0 0 0 0 rgba(0,0,0,0)` for hidden shadow

Reply:
But probably better to filter on hidden field

Reply:
done changed it to filter with `hidden` flag now